src/modules/launcher/PowerLauncher/MainWindow.xaml.cs

```diff
@@ -209,10 +180,37 @@ private void WindowsXamlHostTextBox_ChildChanged(object sender, EventArgs ev)
                         if (!_viewModel.LastQuerySelected)
                         {
                             _viewModel.LastQuerySelected = true;
-                        }
+                        }                        
                     }
                 }
-            };
+            };           
+        }
+
+        private void ApplyTheme (Color theme)
+        {
+            if (theme == LightTheme)
+            {               
+                this.SearchBoxBorder.BorderBrush = BorderBrushLightTheme;
+                this.ListBoxBorder.BorderBrush = BorderBrushLightTheme;
+            }
+            else
+            {
+                this.SearchBoxBorder.BorderBrush = BorderBrushDefaultTheme;
+                this.ListBoxBorder.BorderBrush = BorderBrushDefaultTheme;
+            }

```

Comment:
Is applying the brush here a requirement to punch through the xaml island? Or could we target these in a xaml [Resource Dictionary ](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/resourcedictionary-and-xaml-resource-references)

Reply:
@ryanbodrug-microsoft I think your question is similar to Alekhay's. I tried putting them in a resource file and use `dynamic binding` but it didn't update the UI.